contracts/BaseStrategy.sol

```diff
@@ -769,25 +766,6 @@ abstract contract BaseStrategy {
         emit EmergencyExitEnabled();
     }
 
-    /**
-     * Override this to add all tokens/tokenized positions this contract
-     * manages on a *persistent* basis (e.g. not just for swapping back to
-     * want ephemerally).
-     *
-     * NOTE: Do *not* include `want`, already included in `sweep` below.
-     *
-     * Example:
-     *
-     *    function protectedTokens() internal override view returns (address[] memory) {
-     *      address[] memory protected = new address[](3);
-     *      protected[0] = tokenA;
-     *      protected[1] = tokenB;
-     *      protected[2] = tokenC;
-     *      return protected;
-     *    }
-     */
-    function protectedTokens() internal virtual view returns (address[] memory);
```

Comment:
Why is this gone?

Reply:
PR is based on top of https://github.com/yearn/yearn-vaults/issues/241

Reply:
that's an issue not a PR?

Reply:
Either way, make this suggestion a separate PR